authBackEnd/apps/dashboard/urls2.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path, include, re_path
+from .views import DashboardView, AppView
+
+urlpatterns = [
+    re_path(r'^.*$', AppView.as_view(), name='app'),
+]
```

Comment:
why are there two url files? they should be combined

Reply:
completed